TDDStrictequal.js

```diff
@@ -1,5 +1,5 @@
 const strictEquals = (valNumOne, valNumTwo) => {
```

Comment:
`valOne` is a good name, `num` implies they are always numbers which they aren't.